Package.swift

```diff
@@ -1,10 +1,10 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 5.9
 import PackageDescription
 
 let package = Package(
     name: "DualCameraKit",
     platforms: [
-        .iOS(.v18)
```

Comment:
iOS 18 not supported by `// swift-tools-version: 5.9` update. really iOS 17 is the minimum we want anyway, no changes needed to accomodate this lower version